src/main/java/comp3111/covid/Controller.java

```diff
@@ -124,6 +122,10 @@ public void initialize() {
 		DataAnalysis.initCountriesDict(iDataset);
 		List<String> countries = new ArrayList<String>(DataAnalysis.countriesDict.values());
 		Collections.sort(countries);
+		for (String prioritizedCountry: Arrays.asList( "World", "United Kingdom", "Singapore", "Macao", "Japan", "Israel", "India", "Hong Kong")){
```

Comment:
Please make the list as a `final private` variable in the `Controller `class, and use reversed sorted for loop to insert the list